dash/testing/wait.py

```diff
@@ -62,8 +63,9 @@ def __call__(self, driver):
         try:
             elem = driver.find_element(By.CSS_SELECTOR, self.selector)
             logger.debug("contains text {%s} => expected %s", elem.text, self.text)
-            return self.text in str(elem.text) or self.text in str(
-                elem.get_attribute("value")
+            value = elem.get_attribute("value")
+            return str(elem.text) in self.text or (
```

Comment:
This is swapped -- should be `self.text in str(elem.text)` and `self.text in str(value)`

Reply:
Thanks. Got confused on the order. Fixed.